core/field_colour.js

```diff
@@ -78,15 +78,15 @@ Blockly.FieldColour.fromJson = function(options) {
  * @private
  * @const
  */
-Blockly.FieldColour.DEFAULT_WIDTH = 16;
+Blockly.FieldColour.DEFAULT_WIDTH = 16 + Blockly.Field.X_PADDING;
 
 /**
  * Default height of a colour field.
  * @type {number}
  * @private
  * @const
  */
-Blockly.FieldColour.DEFAULT_HEIGHT = 12;
+Blockly.FieldColour.DEFAULT_HEIGHT = Blockly.Field.BORDER_RECT_DEFAULT_HEIGHT;
```

Comment:
Is losing the -1 intentional?

Reply:
Yes.  The field was reporting its height to be smaller than its actual bounds, in order to make the spacer below it smaller.  I don't want to do that--the height should be accurate--so I'm allowing it to have a slightly taller spacer below.